dojo/api_v2/serializers.py

```diff
@@ -1202,7 +1202,7 @@ def get_findings_list(self, obj) -> List[int]:
 
 
 class ImportScanSerializer(serializers.Serializer):
-    scan_date = serializers.DateField(default=datetime.date.today)
+    scan_date = serializers.DateField()
```

Comment:
The `scan_date` is still a mandatory field. This doesn't fit to the documentation. If it should be optional, I think you need to set `required=False`, see https://www.django-rest-framework.org/api-guide/fields/#required